src/context.js

```diff
@@ -17,20 +16,23 @@ const helpers = {
     return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
   },
   inflection,
-  changeCase
+  changeCase,
 }
 
 const localsToCapitalize = ['name']
 const localsDefaults = {
-  name: 'unnamed'
+  name: 'unnamed',
 }
+
 const capitalizedLocals = (locals: any) =>
-  L.mapValues(
-    L.mapKeys(L.pick(locals, localsToCapitalize), (v, k) =>
-      helpers.capitalize(k)
-    ),
-    v => helpers.capitalize(v)
-  )
+  Object.entries(locals).reduce((hsh, [key, value]) => {
+    hsh[key] = value
+
+    if (localsToCapitalize.includes(key))
+      hsh[helpers.capitalize(key)] = helpers.capitalize(value)
+
+    return hsh
+  }, {})
```

Comment:
I'm wondering, if here you can create some kind of higher order function locally, like `filter` in this portion of the code:

https://github.com/jondot/hygen/blob/b2925f1eafa63bc73175ecba7122b99a52f790ec/src/render.js#L16

in this case the `mapValues`, `pick`, and `mapKeys` kind of contributed to readability.